tests/unit/dojson/test_dojson_conferences.py

```diff
@@ -565,40 +565,50 @@ def test_series_from_411__a():
         '</datafield>'
     )  # record/1430017
 
-    expected = [
-        'DPF Series',
-    ]
     result = clean_record(conferences.do(create_record(snippet)))
 
-    assert expected == result['series']
+    assert result['series'] == [
+        {
+            'name': 'DPF Series'
+        },
+    ]
 
 
 def test_series_number_from_411__n():
     snippet = (
-        '<datafield tag="411" ind1=" " ind2=" ">'
-        '  <subfield code="n">7</subfield>'
-        '</datafield>'
+        '<record>'
+        '   <datafield tag="411" ind1=" " ind2=" ">'
+        '       <subfield code="n">7</subfield>'
+        '   </datafield>'
+        '</record>'
     )  # record/1447029
 
-    expected = 7
     result = clean_record(conferences.do(create_record(snippet)))
 
-    assert expected == result['series_number']
+    assert result['series'] == [
+        {
+            'number': 7
+        },
+    ]
```

Comment:
Better to put this in an `expected` variable, so that you can consistently write `expected == result[...]`, so `AssertionError`s always have the same thing on left and right and can be read more consistently.